### PR TITLE
Update linux.md fedora install instructions

### PR DESCRIPTION
**Pull request by grecko123**
Add essential step of reloading the firewall rules, so as to apply the newly loaded changes.

### CODE REVIEW

File: docs/general/installation/linux.md [hunk 1]
```diff
@@ -96,7 +96,13 @@ However [`rpmfusion`](https://rpmfusion.org/) provides both `jellyfin-server` an
 
    :::
 
-6. Go to `localhost:8096` or `ip-address-of-jellyfin-server:8096` to finish setup in the web UI
+6. Reload the firewall to apply the new rules:
+
+   ```sh
+   sudo firewall-cmd --list-services
```
Comment (felix920506) [inline docs/general/installation/linux.md:102]: I don't think this is the right command for reloading firewalld

**Review by felix920506 — CHANGES_REQUESTED** [review]

File: docs/general/installation/linux.md (hunk 1 above)
Comment (Shadowghost) [reply docs/general/installation/linux.md:102]: Correct one should be `sudo firewall-cmd --reload`

**Comment by joshuaboniface** [comment]
So I want to jump in this one and suggest, that we also update it to leverage the RPMFusion packages/repo instead of our, since the latter goes away with 10.9.0.

**Comment by sigaloid** [comment]
+1, this blocked me from setting it up for a little while

File: docs/general/installation/linux.md (hunk 1 above)
Comment (grecko123) [reply docs/general/installation/linux.md:102]: wow- i do know that :joy:  

and i dont know how I pushed this, must have been in a rush

**Review by felix920506 — APPROVED** [review]

**Pull request merged**